.github/workflows/javascript-test-feature-branch.yml

```diff
@@ -0,0 +1,52 @@
+name: Test feature branch
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        required: false
+        default: "."
+        type: string
+        description: directory to run all run commands from
+
+jobs:
+  quality:
+    defaults:
+      run:
+        working-directory: ${{ inputs.working-directory }}
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, 'wip:') }}
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: configure-node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - name: Get node version
+        id: node
+        run: |
+          echo "version=$(node -v)" >> $GITHUB_OUTPUT
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
+      - name: Cache node modules
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}-${{ steps.node.outputs.version }}
+      - name: create .yarnrc file
+        run: |
+          npm config set "@telicent-io:registry" "https://TODO/"
+          npm config set "//TODO/:_authToken" ${{ secrets.NPMJS_AUTH_TOKEN }} 
+          npm config set "@fortawesome:registry" https://npm.fontawesome.com/
+          npm config set  "//npm.fontawesome.com/:_authToken" ${{ secrets.FONT_AWESOME_AUTH_TOKEN }}
```

Comment:
For the public stuff I dont think we will need this font awesome auth token and  config 
```suggestion
          
```

Reply:
@davec504 is this correct?